test/fixtures/e2e/amp-bind/bind-amp4email.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
```

Comment:
Are these new fixtures a straight copy paste from `test/fixtures`? Maybe we can remove the old ones or rename the existing ones to try to keep the history.

Reply:
They are not a direct copy-paste, but there's really only one difference since these tests have their extension script paths as `cdn.ampproject.org` and the others reference the extensions from the `/dist` folder

Reply:
I think I can move them, actually since it looks like my assumption that these tests are used somewhere else is not true.